src/__tests__/__snapshots__/Label.js.snap

```diff
@@ -8,6 +8,8 @@ exports[`Label respects the "outline" prop 1`] = `
   color: #fff;
   border-radius: 2px;
   background-color: #6a737d;
+  width: medium;
+  height: medium;
```

Comment:
Good question - I'm not sure. It's happening because we've got the `size` prop from styled system and a `size` prop in this component that corresponds to a size map that sets padding and font size. I think maybe the size prop in styled system wasn't working properly before and we never noticed it? 

To work around this I tried removing `size` from the `LAYOUT` group, but there's a bug in styled-system v5 that makes it so that if you import just one prop from the layout group, it imports all of the props. So for now I'm just going to leave it as is. It sucks that we have these invalid CSS rules being added but there's not much we can do until that bug gets fixed in styled system. Might look into fixing it myself if I can find the time